app/src/main/java/com/github/se/project/ui/authentification/TutorInfoScreen.kt

```diff
@@ -95,7 +95,7 @@ fun TutorInfo(profile: Profile) {
                 if (algebreChecked.value) {
                   Icon(Icons.Filled.Check, contentDescription = null)
                 }
-                Text("Algèbre")
+                Text("AlgÃ¨bre")
```

Comment:
The text is currently written in French, but the app's content is primarily in English. Maybe you should update the text to maintain consistency.
Also for Analyse & Phisique

Reply:
That's true but as the courses names are in French, I think it's okay